index.ts

```diff
@@ -29,28 +31,43 @@ server.use(restify.gzipResponse());
 server.use(restify.fullResponse());
 server.use(restify.throttle(conf.get('throttleOptions')));
 server.use(restify.requestLogger());
-server.use(restify.queryParser());
+server.use(restify.queryParser({ mapParams: false }));
+server.use(validator());
 server.use(util.getCert);
 server.use(util.log);
 server.use(blpSession.getSession);
 server.use(requestHandler.elevateRequest);
 
-// Route
-server.post('/request/:ns/:svName/:reqName',
+// Routing
+
+// `/request` handles request/response services. There is one operation on these services, namely
+// "get me some data". In some sense this ought to be a GET but because the request data is large
+// we put it in the body.
+server.post('/request',
+            util.validateRequest((req: Interface.IOurRequest): void => {
```

Comment:
I like having the param values in `index.ts` because then you can see in one place what all the query params ought to be, but I did create some wrappers to make this a lot clearer.